msteams-platform/m365-apps/extend-m365-teams-message-extension.md

```diff
@@ -69,7 +69,15 @@ To start with a [sample message extension](https://github.com/OfficeDev/TeamsFx-
 
     :::image type="content" source="images/toolkit-palatte-search-sample.png" alt-text="Screenshot shows the Create a new Teams app VS Code command palette to list Teams sample options.":::
 
-    The sample is also available as *NPM Search Connector* in the Teams Toolkit Samples gallery. From the Teams Toolkit pane, select **Development** > **View samples** > **NPM Search Connector**.
+1. Select **Custom Search Results** to download the sample code for a Teams message extension using the latest Teams app manifest. For more information, see [Teams developer manifest](../resources/schema/manifest-schema.md).
```

Comment:
Ah! That's great there's a M365 ME sample ("Custom Search Results") directly in the Toolkit wizard for creating a new app. We should definitely use it instead of the previous "NPM Search Results" from samples gallery. Could you rework the steps in this Quickstart section to match the flow for this new sample (via "Create a New App" wizard)?

Reply:
Nevermind! Its the same sample :p

Reply:
the steps still hold up and are accurate. 👍